mollie/api/resources/orders.py

```diff
@@ -0,0 +1,32 @@
+from ..error import IdentifierError
+from ..objects.order import Order
+from .base import Base
+
+
+class Orders(Base):
+    RESOURCE_ID_PREFIX = 'ord_'
+
+    def get_resource_object(self, result):
+        return Order(result, client=self.client)
+
+    def get(self, order_id, **params):
+        if not order_id or not order_id.startswith(self.RESOURCE_ID_PREFIX):
+            raise IdentifierError(
+                "Invalid order ID: '{id}'. An order ID should start with '{prefix}'.".format(
+                    id=order_id, prefix=self.RESOURCE_ID_PREFIX)
+            )
+        return super(Orders, self).get(order_id, **params)
+
```

Comment:
I suppose the `updateOrder` and `cancelOrder` call could also benefit from the identifier check?

Maybe the identifier check logic can even be generalized and added to the resource `Base` class, using `RESOURCE_ID_PREFIX` and the `Base.get_resource_name` method.

Reply:
Yes it should, but not as part of this sprint.